src/tree-menu/tree-menu-item.component.ts

```diff
@@ -49,6 +53,17 @@ export class TreeMenuItemComponent extends AbstractTrackerComponent implements O
 
   ngOnInit() {
     this.href = `${this.windowHrefService.getHrefWithoutHash()}#${this.path}`;
+
+    if (this.schema['type'] === 'object') {
+      this.keys = Object.keys(this.value);
+    }
+  }
+
+  ngOnChanges(changes: SimpleChanges) {
```

Comment:
I see that the tree map only seems to realise about top level changes. So if there is author 0 and I add a new one, author 1 appears in the tree map.

But if I had fields to either author, these don't appear in the tree map HTML.